src/ui/public/chrome/directives/header_global_nav/components/header_breadcrumbs.tsx

```diff
@@ -86,4 +83,22 @@ export class HeaderBreadcrumbs extends Component<Props, State> {
       delete this.subscription;
     }
   }
+
+  private getBreadcrumbs() {
```

Comment:
Could get fancy and change this to a get property: `private get breadcrumbs() { }`

Reply:
Sorry, I don't think that's fancy 😝, if there is a function that executes to create something I think it should be a function not hidden behind a magic property that's actually a function in disguise